tests/tests_suite_test.go

```diff
@@ -52,6 +52,7 @@ func TestTests(t *testing.T) {
 
 var _ = BeforeSuite(func() {
 	tests.BeforeTestSuitSetup()
+	tests.AssertDualStackCluster()
```

Comment:
We decide at the meeting to create a test that will fail instead of abort, that give us results but make CI fail.